include/util/hex_float.h

```diff
@@ -711,13 +711,37 @@ std::ostream& operator<<(std::ostream& os, const HexFloat<T, Traits>& value) {
   return os;
 }
 
+// Returns true if the negate_value is true and the next character on the
```

Comment:
Returns true if the negate_value parameter is true 
or 
Returns true if negate_value is true.